### PR TITLE
Use the new cockpit-path helper from cockpit-lib

### PR DESCRIPTION
**Pull request by jelly**

### CODE REVIEW

File: src/components/vm/snapshots/vmSnapshotsCreateModal.jsx [hunk 1]
```diff
@@ -30,9 +30,11 @@ import { DialogsContext } from 'dialogs.jsx';
 import { ModalError } from "cockpit-components-inline-notification.jsx";
 import { FileAutoComplete } from "cockpit-components-file-autocomplete.jsx";
 import { snapshotCreate, snapshotGetAll } from "../../../libvirtApi/snapshot.js";
-import { getSortedBootOrderDevices, LIBVIRT_SYSTEM_CONNECTION, dirname } from "../../../helpers.js";
+import { getSortedBootOrderDevices, LIBVIRT_SYSTEM_CONNECTION } from "../../../helpers.js";
 import { domainGet } from '../../../libvirtApi/domain.js';
 
+import { dirname } from "cockpit-path";
```
Comment (martinpitt) [inline src/components/vm/snapshots/vmSnapshotsCreateModal.jsx:36]: Wrong sorting.. this should move up to the other cockpit lib imports, and then there should be a newline before the c-machines local imports.

**Review by martinpitt — APPROVED** [review]
Thanks! Comment isn't important, I leave this up to you if you bother.

File: src/components/vm/snapshots/vmSnapshotsCreateModal.jsx (hunk 1 above)
Comment (jelly) [reply src/components/vm/snapshots/vmSnapshotsCreateModal.jsx:36]: I can fix that up, I got too used to our files eslint automatically fixing the import grouping

**Review by martinpitt — APPROVED** [review]
Dankjewel!

**Pull request merged**